test-resources/test/test-util/custom-de-serializers.ts

```diff
@@ -0,0 +1,9 @@
+export const customStringPropertyValue = 100;
+
+export const customTestDeSerializers = {
```

Comment:
can you re-use this in the type-tests where I defined a custom deserializer a few time. In batch type test for sure but also in de-serializer type tests

Reply:
done